### PR TITLE
Update: Remove all `this.sendActions`s

### PR DESCRIPTION
**Pull request by Alex-Aralis**
- `this.sendAction('myAction', myValue) ` ==> `this.get('myAction')(myValue)`
- renamed actions with `on`
- explicitly pass action handlers

### CODE REVIEW

File: packages/core/addon/components/pick-object-container.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright 2017, Yahoo Holdings Inc.
+ * Copyright 2019, Yahoo Holdings Inc.
```
Comment (balajimanoharan) [inline packages/core/addon/components/pick-object-container.js]: Can we revert the changes in this file?

File: packages/core/addon/components/pick-container.js [hunk 2]
```diff
@@ -151,14 +161,15 @@ export default Ember.Component.extend({
 
   /**
    * @method toggleForm - toggles pick-form between open and closed states
-   * Sends `formToggled` action with state to parents
+   * Calls onFormToggled handler.
    */
   toggleForm() {
     this.toggleProperty('isFormOpen');
     this._updateFormVisibility();
+    const handleFormToggled = get(this, 'onFormToggled');
```
Comment (balajimanoharan) [inline packages/core/addon/components/pick-container.js]: Can we do this instead to avoid the `if` conditional below?
```
const handleFormToggled = get(this, 'onFormToggled') || (() => {});
```

File: packages/core/addon/components/pick-container.js [hunk 3]
```diff
@@ -13,6 +21,8 @@
 import Ember from 'ember';
 import Layout from '../templates/components/pick-container';
 
+const { get, computed } = Ember;
```
Comment (jkusa) [inline packages/core/addon/components/pick-container.js]: `import { computed, get } from '@ember/object';`

File: packages/reports/addon/templates/components/navi-date-range-picker.hbs [hunk 4]
```diff
@@ -1,5 +1,5 @@
-{{!-- Copyright 2017, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
-{{#pick-container updateSelection=this.attrs.setInterval as |selection container|}}
+{{!-- Copyright 2019, Yahoo Holdings Inc. Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms. --}}
+{{#pick-container onUpdateSelection=this.attrs.onSetInterval as |selection container|}}
```
Comment (jkusa) [inline packages/reports/addon/templates/components/navi-date-range-picker.hbs]: Since we are touching this, can we remove all the `this.attrs`? It is now a private API

File: packages/core/addon/components/pick-container.js (hunk 3 above)
Comment (Alex-Aralis) [reply packages/core/addon/components/pick-container.js]: Odd, i don't even remember adding this import o_O

**Comment by mattgraham1995** [comment]
Is this ready to be merged?
